docs/framework-takipi_agent.md

```diff
@@ -0,0 +1,30 @@
+# Takipi Agent Framework
+The Takipi Agent Framework causes an application to be automatically configured to work with [OverOps Service][].
+
+## Configuration
+For general information on configuring the buildpack, including how to specify configuration values through environment variables, refer to [Configuration and Extension][].
+
+The framework can be configured by modifying the [`config/takipi_agent.yml`][] file. The framework uses the [`Repository` utility support][repositories] and so it supports the [version syntax][] defined there.
+
+| Name | Description
+| ---- | -----------
+| `node_name_prefix` | Node name prefix, will be concatenated with `-` and instance index to form the host name in Takipi.
+| `secret_key` | Installation key to use with SaaS (Optional)
+| `collector_host` | Remote collector hostname (or ip) to use (Optional)
+| `collector_port` | Remote collector port to use (Optional)
+
+The Takipi framework will be activated if either `secret_key` or `collector_host` is set (refer to [OverOps Remote Collector][] for more details) - this can be done by changing the value in `config/takipi_agent.yml` or by setting the environment variable `JBP_CONFIG_TAKIPI_AGENT` as explained in the configuration section.
+
+## Logs
+
+Currently, you can get the Takipi agent logs using `cf files` command:
```

Comment:
Those should be fine as they only need to live for the lifetime of the container itself.  Logs are particularly problematic because you typically need them after something has gone wrong.  And when things go wrong, the container is recycled and the filesystem is disposed of.  This is why all logging is collected from the console and streamed out to `cf logs` or a log warehouse like PaperTrail or Splunk.